pypeit/slittrace.py

```diff
@@ -308,6 +308,24 @@ def spatid_to_zero(self, spat_id):
         """
         return np.where(self.spat_id == spat_id)[0][0]
 
+    def slitord_to_zero(self, slitord):
```

Comment:
As far as I can tell, this is only used in a loop.  There's a way to do this in one go without the loop that should be more efficient, but not sure it's worth the effort.

Reply:
I can imagine it being used outside of a loop.  
Leaving as is for now.